### ISSUE TITLE
DSX Desktop v1.2.3 Install Fails

### ISSUE THREAD

**Issue by cgoo4** [IBMDataScience/DSx-Desktop#64]
<img width="630" alt="docker" src="https://user-images.githubusercontent.com/32419319/38725019-dac240d8-3efd-11e8-9845-bca28d9eca0e.png">
Tried installing v1.2.3 on Windows 7 to see if it resolves previously reported issues. The install fails with "Error Log - Had trouble starting ibm-dsx docker machine". Details below:

```
--
[2018-04-13 09:07:11:0739] [error] exec error: Error: Command failed: docker-machine start ibm-dsx
Error setting up host only network on machine start: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm ibm-dsx --nic2 hostonly --nictype2 82540EM --nicpromisc2 deny --hostonlyadapter2 VirtualBox Host-Only Ethernet Adapter #2 --cableconnected2 on failed:
VBoxManage.exe: error: Code E_FAIL (0x80004005) - Unspecified error (extended info not available)
VBoxManage.exe: error: Context: "LockMachine(a->session, LockType_Write)" at line 525 of file VBoxManageModifyVM.cpp
 
[2018-04-13 09:07:11:0739] [error] stderr: Error setting up host only network on machine start: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm ibm-dsx --nic2 hostonly --nictype2 82540EM --nicpromisc2 deny --hostonlyadapter2 VirtualBox Host-Only Ethernet Adapter #2 --cableconnected2 on failed:
VBoxManage.exe: error: Code E_FAIL (0x80004005) - Unspecified error (extended info not available)
VBoxManage.exe: error: Context: "LockMachine(a->session, LockType_Write)" at line 525 of file VBoxManageModifyVM.cpp
 
[2018-04-13 09:07:11:0739] [error] Had trouble starting ibm-dsx docker machine

```

**Comment by jerry1100** [comment]
Hi @cgoo4, thanks for reporting this issue.

Based off of this line:
```
VBoxManage.exe: error: Context: "LockMachine(a->session, LockType_Write)" at line 525 of file VBoxManageModifyVM.cpp
```
it seems like the VM is locked, preventing any changes. We can try to remove the VM and start over.

### Steps
1. Open VirtualBox and verify that there is a VM named `ibm-dsx`
2. Open a command prompt and enter `docker-machine rm -y ibm-dsx`
3. Open VirtualBox again and the `ibm-dsx` VM should be gone
4. Try to install DSX Desktop again

If that doesn't work, could you provide your:
1. Log files (with the DSX Desktop open, in the menu: Help > Open log folder > dsx-desktop.log)
2. Docker Toolbox version (open command prompt and enter `docker -v`)
3. VirtualBox version (with VirtualBox open, in the menu: VirtualBox > About VirtualBox)

Thanks.

**Comment by cgoo4** [comment]
Jerry hi - I tried steps 1-4.  I could verify ibm-dsx. After step 2, VM was gone. The install then failed. I've attached the log. Kind regards,

Docker version 18.02.0-ce, build fc4de447b5
VB version 5.2.8 r121009 (Qt5.6.2)
[dsx-desktop.log](https://github.com/IBMDataScience/DSx-Desktop/files/1932758/dsx-desktop.log)

**Comment by cgoo4** [comment]
After doing these three things it installed at the second attempt:

1. `docker-machine rm -y ibm-dsx`
2. Disable network threat protection
3. Run as administrator

The first attempt got stuck at 86%. The second worked.

I'm still unable to load the `lubridate` and other packages per #57.  Please could you comment on that one?

**Comment by cgoo4** [comment]
Hi Jerry - yes, it is Symantec Endpoint Protection, and unfortunately it doesn't allow itself to stay disabled (rightly really) for more than a few minutes, so it's possibly "hit and miss" as to whether it's off at the crucial point.

**Comment by jerry1100** [comment]
I see. I just tried to install v1.2.3 with Symantec enabled and it failed (albeit with a different error). Disabling Symantec allowed it to finish.

Interestingly enough, when I was testing the latest internal build, the installation completed even with Symantec enabled. I don't know what changed that fixed it, but the behavior is consistent. That said, it might be worth waiting until we release v1.2.4 (sometime this week) to see if your error goes away with that version.

**Comment by jerry1100** [comment]
Hi @cgoo4, we just released v1.2.4. Please give that a try and let me know if it works.

**Comment by cgoo4** [comment]
Thanks. Resolved with v1.2.4.

**Comment by jerry1100** [comment]
Glad to hear. :)
